tests/testthat.R

```diff
@@ -0,0 +1,5 @@
+library(testthat)
+
+pkg_name <- "scda"
+library(pkg_name, character.only = TRUE)
```

Comment:
`"only" can weaken meaning` ahah I just found this funny. @cicdguy I remember a discussion about grammar checks, is there any update?